new-components/enrichers/custom-annotation/.env

```diff
@@ -0,0 +1,5 @@
+SMITHY_INSTANCE_ID=8d719c1c-c569-4078-87b3-4951bd4012ee
+SMITHY_LOG_LEVEL=debug
+SMITHY_BACKEND_STORE_TYPE=local
+CUSTOM_ANNOTATION_NAME=reachability
+CUSTOM_ANNOTATION_VALUES={"foo":"bar"}
```

Comment:
this is used only for tests, correct? does it make sense to be in a dedicated dir or have a relevant comment 
so it's clear how the example components work?

Reply:
Done with comments